airflow/providers/amazon/aws/triggers/s3.py

```diff
@@ -39,7 +39,9 @@ class S3KeyTrigger(BaseTrigger):
     :param wildcard_match: whether the bucket_key should be interpreted as a
         Unix wildcard pattern
     :param aws_conn_id: reference to the s3 connection
+    :param use_regex: whether to use regex to check bucket
     :param hook_params: params for hook its optional
+    :param use_regex: whether to use regex to check bucket
```

Comment:
added twice?

Reply:
Yep, it's accidentally added 🤦‍♂️ Just remove it. Thanks!